### PR TITLE
Implement LoaderTrait for slices

### PR DESCRIPTION
**Pull request by billy1624**
## New Features

- [x] Implement `LoaderTrait` for slices: `[M; N]` and `&[M]`

### CODE REVIEW

File: src/query/loader.rs [hunk 1]
```diff
@@ -32,8 +32,63 @@ pub trait LoaderTrait {
...
+        R::Model: Send + Sync,
+        <<Self as LoaderTrait>::Model as ModelTrait>::Entity: Related<R>,
+    {
+        self.as_slice().load_many(stmt, db).await
+    }
+}
+
+#[async_trait]
+impl<M, const N: usize> LoaderTrait for [M; N]
+where
```
Comment (tyt2y3) [inline src/query/loader.rs:62]: I don't think we needed this, as it's just trivial casting `[M; N]` to `&[M]`?

Comment (billy1624) [reply src/query/loader.rs:62]: Yes, it's trivial to cast it. But wouldn't it be awesome we can write... directly?

```rs
let fruits = [fruit::Model {
    id: 1,
    name: "Apple".to_owned(),
    cake_id: Some(1),
}];

let cakes = fruits
    .load_one(cake::Entity::find(), &db)
    .await?;
```

Comment (tyt2y3) [reply src/query/loader.rs:62]: In your example it think it will "auto as ref" so `as_slice` is not needed?

Comment (billy1624) [reply src/query/loader.rs:62]: I tested it. It said "`[M; N]` does not implement ..."
So, it won't "auto as ref".

Comment (tyt2y3) [reply src/query/loader.rs:62]: Ah sad, but I still don't like it.

Comment (billy1624) [reply src/query/loader.rs:62]: I'll remove it then :(

**Pull request merged**